packager/templates/debian/legacy/server.service

```diff
@@ -3,8 +3,6 @@ Description=The Choria Orchestrator Server
 After=network.target
 
 [Service]
-StandardOutput=syslog
-StandardError=syslog
```

Comment:
the `legacy` package is for circa 2016 ubuntu, so we need to be sure this is fine, @smortex might know

Reply:
I did this in 0f0f91bb3ce4057e4773e4b6a8fda6a132ea68a9 part of #1388 and which I need to rebase on master :-)

It was already obsolete back at this time, just not warning about it.

Reply:
Ah crap, this PR is also not up-to-date with the base branch, I was ready to merge this one before rebasing my changes.

Reply:
@bastelfreak I removed it from my branch since you address all platforms and I only checked the Debian based ones :-)

Reply:
I rebased it